form/create_custom_field_type.rst

```diff
@@ -67,14 +67,31 @@ check out the `ChoiceType`_ class.
     extend from ``AbstractType`` is only a convenience way of implementing the
     required ``FormTypeInterface``.
 
-There are three methods that are particularly important:
+There are a few methods that are particularly important:
 
 .. _form-type-methods-explanation:
 
+``getParent()``
+    This return the class name of the type configuring the form before. It
```

Comment:
returns

Reply:
I am not really sure if I completely understand what the meaning of the "the form before" part is supposed to be.